modules/rn_3_80.adoc

```diff
@@ -0,0 +1,148 @@
+
+////
+
+* The following configuration fields have been added to enhance the super user role: 
+
+** **LDAP_SUPERUSER_FILTER**: This configuration field is a subset of the `LDAP_USER_FILTER` configuration field. It allows {productname} administrators the ability to configure Lightweight Directory Access Protocol (LDAP) users as super users when {productname} uses LDAP as its authentication provider.
++
+With this field, administrators can add or remove super users without having to update the {productname} configuration file and restart their deployment. 
+
+** **FEATURE_SUPERUSERS_FULL_ACCESS**: This configuration field grants super users the ability to read, write, and delete content from other repositories in namespaces that they do not own or have explicit permissions for. 
+
+* The following configuration fields have been added for user permissions: 
+
+** **FEATURE_RESTRICTED_USERS**: This configuration field restricts normal users from creating organizations and reading and writing content. 
+
+** **RESTRICTED_USERS_WHITELIST**: With this configuration field enabled, administrators can exclude users from the `FEATURE_RESTRICTED_USERS` setting. 
+
+
+////
+* The following enhancements have been made to user permissions:
+
+** Previously, {productname} did not allow global access for any user, including super users. This update adds the ability to create users with global read-only access. Users with read-only access can see all repositories and image across any namespace and any organization. For more information, see link:https://issues.redhat.com/browse/PROJQUAY-2604[PROJQUAY-2604]. 
+
+* The following changes have been made to user permissions:
+
+** {productname} administrators can now limit normal users from creating organization in {productname} and assign this option to specific users. For more information, see link:https://issues.redhat.com/browse/PROJQUAY-402[PROJQUAY-402]. 
+////
+
+
+Known issues:
+
+=== {productname} Operator 
+
+Added/Changed:
+
+* IPv6 support is now available in {productname} {producty} Operator deployments. With this feature, the {productname} Operator can now be deployed in IPv4 and IPv6 dual-stack OpenShift Container Platform clusters where {productname} uses IPv6.  
+
+
+=== {productname} feature tracker
+
+
+
+New features have been added to {productname}, some of which are currently in Technology Preview. Technology Preview features are experimental features and are not intended for production use.
+
+Some features available in previous releases have been deprecated or removed. Deprecated functionality is still included in {productname}, but is planned for removal in a future release and is not recommended for new deployments. For the most recent list of deprecated and removed functionality in {productname}, refer to the table below. Additional details for more fine-grained functionality that has been deprecated and removed are listed after the table.
```

Comment:
"refer to the table below."

I think you could just say "refer to Table 1.1." to avoid using directional language.
SSG https://redhat-documentation.github.io/supplementary-style-guide/#cloud-services-visual-info